src/types.ts

```diff
@@ -117,7 +117,7 @@ export interface Section {
 export interface Question {
   id: string;
   label?: string;
-  value?:string;
+  value?: string;
   type: string;
```

Comment:
We should make this optional since markdown content doesn't have a type

Reply:
And we should update this in form engine too

Reply:
@NethmiRodrigo , we have a PR here [feat: FormField Type should be Optioanl #419](https://github.com/openmrs/openmrs-esm-form-engine-lib/pull/419) for this.